contracts/src/NativeTokenDestination.sol

```diff
@@ -118,61 +127,54 @@ contract NativeTokenDestination is
     uint256 public lastestBurnedFeesReported;
 
     /**
-     * @notice The wrapped native token contract that represents the native tokens on this chain.
+     * @dev When modifier is used, the function can only be called after the contract is fully collelateralized,
```

Comment:
I went for simply "The native minter precompile." because with the address specified I think it's technically referring to an instance of the interface, but didn't want to overcomplicate things.